calico/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - calico
 ## 1.0.0 / 2022-05-06
+## 1.0.1 / 2022-05-18
+
+* [Fixed] Fix extra metrics description example. See [#12043](https://github.com/DataDog/integrations-core/pull/12043).
+
```

Comment:
Looks like the formatting is messed up here

Reply:
Good catch, updated!

Reply:
I think the spacing was off beforehand: https://github.com/DataDog/integrations-core/pull/11959/commits/c30b629d1e7a5d54ec30d44bed1501fee85b59d2